projects/canopy/src/lib/card/docs/promotions/guide.stories.mdx

```diff
@@ -0,0 +1,513 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import Feedback from '../../../docs/common/feedback.md';
+
+<Meta title="Patterns/Promotions/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Promotions
+
+<p class="standfirst">Promotions are containers for content and actions on a specific topic, that use the Cards component and combine the Grid functionality to obtain a Magazin layout based on how many promotions you want to display.</p>
+
+## Usage
+
+They are used to group promotional content within a page.
+
+Import the card module in your application:
+
+```js
+@NgModule({
+  ...
+  imports: [ ..., LgCardModule ],
+});
+```
+
+and in your HTML, for a general promotion card with image:
+
+```html
+<lg-card
+  lgShadow
+  lgMarginBottom="lg"
+  lgPadding="none"
+  variant="promotion"
+  [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'horizontal' }">
+  <lg-card-hero-img [coverImageUrl]="imageUrl">
+  </lg-card-hero-img>
+  <lg-card-content>
+    <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+    <p lgMarginBottom="lg">{{ text }}</p>
+    <a href="#">{{ buttonText }}</a>
+  </lg-card-content>
+</lg-card>
+```
+
+and in your HTML, for a general promotion card with icon:
+
+```html
+<lg-card
+  lgShadow
+  lgMarginBottom="lg"
+  variant="promotion"
+  [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'horizontal' }">
+  <lg-card-hero-img>
+    <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+  </lg-card-hero-img>
+  <lg-card-content>
+    <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+    <p lgMarginBottom="lg">{{ text }}</p>
+    <a href="#">{{ buttonText }}</a>
+  </lg-card-content>
+</lg-card>
+```
+
+## How it works
+
+Canopy provides a number of patterns for a range of common scenarios, and we recommend you use those before creating something from scratch.
+1. **Stand-alone Card** pattern layout.
+2. **Two Cards** pattern layout.
+3. **Three Cards** pattern layout.
+4. **Magazine** pattern layout.
+
+## Dos and Don'ts
+
+### Do
+
+1. **Do** keep the message succinct.
+2. **Do** use equal amount of content across components for alignment.
+3. **Do** use in line with the variants available.
+
+## Promotions content and templates
+
+As the promotion card is just a container for content, you can add whatever content you like.
+In the examples bellow, we use The Canopy Grid System to get the desired layout and the component `LgCardHeroImageComponent` uses an image but it can be used with either an Image or a Brand Icon.
```

Comment:
I don't think `The` has to be capitalized

Reply:
Done. :)